src/helpers/DirectionalLightHelper.js

```diff
@@ -74,13 +74,13 @@ DirectionalLightHelper.prototype.update = function () {
 		v3.subVectors( v2, v1 );
 
 		this.lightPlane.lookAt( v3 );
-		if ( ! this.overrideColor ) this.lightPlane.material.color.copy( this.light.color );
-		else this.lightPlane.material.color.set( this.overrideColor );
+		if ( ! this.color ) this.lightPlane.material.color.copy( this.light.color );
```

Comment:
I assume you intend for `color` to be hex? What if `color` is `0x000000`? The pattern to use is
```
if ( color !== undefined ) { 

} else {

}
```